doc/api/repl.markdown

```diff
@@ -190,15 +194,59 @@ be emitted.
 Example of listening for `reset`:
 
     // Extend the initial repl context.
-    var r = repl.start({ options ... });
+    var replServer = repl.start({ options ... });
     someExtension.extend(r.context);
 
     // When a new context is created extend it as well.
-    r.on('reset', function (context) {
+    replServer.on('reset', function (context) {
       console.log('repl has a new context');
       someExtension.extend(context);
     });
 
+### replServer.displayPrompt([preserveCursor])
+
+* `preserveCursor` {Boolean}
+
+Like [readline.prompt][] except also adding indents with ellipses when inside
+blocks. The `preserveCursor` argument is passed to [readline.prompt][]. This is
+used primarily with `defineCommand`. It's also used internally to render each
+prompt line.
+
+### replServer.defineCommand(keyword, cmd)
+
+* `keyword` {String}
+* `cmd` {Object|Function}
+
+Makes a command available in the REPL. In the REPL, the command is invoked by
+typing a `.` followed immediately by the keyword. The `cmd` is object with the
+following values:
+
+ - `help` - help text to be displayed when `.help` is entered (Optional).
+ - `action` - a function to execute, potentially taking in a string argument,
```

Comment:
Nit: "potentially taking in a string argument" seems like an unnecessary variant on the more common and perhaps widely understood "It takes an optional string argument" or "It optionally takes a string argument".